src/Agent.PluginHost/Program.cs

```diff
@@ -68,6 +69,13 @@ public static int Main(string[] args)
                         ArgUtil.NotNull(taskPlugin, nameof(taskPlugin));
                         taskPlugin.RunAsync(executionContext, tokenSource.Token).GetAwaiter().GetResult();
                     }
+                    catch (SocketException ex)
+                    {
+                        executionContext.Error("SocketException occurred.");
```

Comment:
Please move duplicated blocks to the separate method.

Reply:
Done. These code blocks were moved to handlers.